src/plugins/telemetry/schema/legacy_plugins.json

```diff
@@ -1,21 +1,3 @@
 {
-  "properties": {
-    "localization": {
-      "properties": {
-        "locale": {
-          "type": "keyword"
-        },
-        "integrities": {
-          "properties": {
-            "DYNAMIC_KEY": {
-              "type": "text"
-            }
-          }
-        },
-        "labelsCount": {
-          "type": "long"
-        }
-      }
-    }
-  }
+  "properties": {}
 }
```

Comment:
@elastic/kibana-telemetry This PR removes the last legacy usage collector. I guess this file and the generation / usage around it could now be safely deleted I guess. Not sure if it is trivial enough to be done in current PR though. Would it be fine to let you handle that in a follow-up?

Reply:
We'd have to update the `.telemetryrc.json` file and it will automatically be removed. We can push that change in another PR